xAPI.md

```diff
@@ -313,21 +312,38 @@ an IRL is an IRI that when translated into a URI (per the IRI to URI rules), is
 <a name="def-inverse-functional-identifier" />
 
 __Inverse Functional Identifier__: An identifier which is unique to a particular persona or group.
- Used to identify Agents and Groups.
+
+<a name="def-learning-experience" />
+
+__Learning Experience__: An event associated with the acquisition of knowledge.  It is highly diverse 
+as far as what it can be.  Examples include reading a book, taking an online course, going on a field trip, 
+engaging in self-directed research, or receiving a certificate for a completed course.
 
 <a name="def-learning-management-system" />
 
-__Learning Management System (LMS)__: "A software package used to administer one or more courses to one or more learners. An LMS is typically a web-based 
-system that allows learners to authenticate themselves, register for courses, complete courses and take 
-assessments" (Learning Systems Architecture Lab definition). In this document the term will be used in the context of 
-existing systems implementing learning standards.
+__Learning Management System (LMS)__: "A software package used to administer one or more courses to one or more learners. 
+An LMS is typically a web-based system that allows learners to authenticate themselves, register for courses, complete 
+courses and take assessments" (Learning Systems Architecture Lab definition). In this document the term will be used in 
+the context of existing systems implementing learning standards.
+
+<a name="def-learning-record" />
+
+__Learning Record__: An account of a Learning Experience that is formatted according to the rules of xAPI.  A Learning Record
+takes on many forms, including Statements, documents, and their parts.  This definition is intended to be all-inclusive.
+
+<a name="def-learning-record-consumer" />
+
+__Learning Record Consumer__: An xAPI Client that accesses data from Learning Record Store(s).
```

Comment:
What I think we need is a term that denotes something that interprets or uses the data in some way. With this definition, I can't think of anywhere we'd use this term instead of client.